src/Domain/Apprenticeship/EpisodeDomainModel.cs

```diff
@@ -0,0 +1,232 @@
+﻿using SFA.DAS.Apprenticeships.DataAccess.Entities.Apprenticeship;
+using SFA.DAS.Apprenticeships.Enums;
+using System.Collections.ObjectModel;
+
+namespace SFA.DAS.Apprenticeships.Domain.Apprenticeship
+{
+    public class EpisodeDomainModel
+    {
+        private readonly Episode _entity;
+        private readonly List<EpisodePriceDomainModel> _episodePrices;
+        public Guid Key => _entity.Key;
+        public long Ukprn => _entity.Ukprn;
+        public long EmployerAccountId => _entity.EmployerAccountId;
+        public FundingType FundingType => _entity.FundingType;
+        public FundingPlatform? FundingPlatform => _entity.FundingPlatform;
+        public long? FundingEmployerAccountId => _entity.FundingEmployerAccountId;
+        public string LegalEntityName => _entity.LegalEntityName;
+        public long? AccountLegalEntityId => _entity.AccountLegalEntityId;
+        public string TrainingCode => _entity.TrainingCode;
+        public string TrainingCourseVersion => _entity.TrainingCourseVersion;
+        public bool PaymentsFrozen => _entity.PaymentsFrozen;
+        public IReadOnlyCollection<EpisodePriceDomainModel> EpisodePrices => new ReadOnlyCollection<EpisodePriceDomainModel>(_episodePrices);
+        public List<EpisodePriceDomainModel> ActiveEpisodePrices => _episodePrices.Where(x => !x.IsDeleted).ToList();
+        public EpisodePriceDomainModel LatestPrice
+        {
+            get
+            {
+                var latestPrice = _episodePrices.Where(y => !y.IsDeleted).MaxBy(x => x.StartDate);
+                if (latestPrice == null)
+                {
+                    throw new InvalidOperationException($"Unexpected error. {nameof(LatestPrice)} could not be found in the {nameof(EpisodeDomainModel)}.");
+                }
+
+                return latestPrice;
+            }
+        }
+
+        public EpisodePriceDomainModel FirstPrice
+        {
+            get
+            {
+                var latestPrice = _episodePrices.Where(y => !y.IsDeleted).MinBy(x => x.StartDate);
```

Comment:
very minor but should this variable be named firstPrice? looks like a copy paste error